src/polytope/github/repository/InternalCode.py

```diff
@@ -0,0 +1,26 @@
+from enum import Enum, auto
+
+
+class GithubRepositoryInternalCode(Enum):
+    """! Internal status code of Github CRUD requests."""
+
+    # Success.
+    Success = auto()
+    # create cannot modify repository name.
+    ForbiddenNameChangeOnCreation = auto()
+    # config validation failed.
+    ConfigValidationFailed = auto()
+    # creation failed on Github side.
+    FailedToCreate = auto()
```

Comment:
(question) Should we distinguish the case of creation failed due to duplication of repositories' names?

Reply:
I'll lease this for future with a todo mark. It'll be efficient to focus on frequent and valuable reasons of failure.